frontend/smart-kickers-game/src/App.js

```diff
@@ -1,42 +1,37 @@
-import { useEffect, useState } from 'react'
-import './App.css'
-import {Button} from "./components/button"
+import { useEffect, useState } from 'react';
+import './App.css';
+import { Button } from './components/button';
 
 function App() {
-
-  let [blueScore, setBlueScore] = useState(0);
-  let [whiteScore, setWhiteScore] = useState(0);
+  const [blueScore, setBlueScore] = useState(0);
+  const [whiteScore, setWhiteScore] = useState(0);
 
   useEffect(() => {
     const socket = new WebSocket('ws://localhost:3006/csc');
-    //const socket = new WebSocket("ws://localhost:3000")
+    
     socket.onopen = function () {
-      console.log("connected");
       //send to server
-      socket.send("Hello from client")
+      socket.send('Hello from client');
       socket.onmessage = (msg) => {
-        console.log(msg);
-        msg = JSON.parse(msg.data)
-        console.log(msg)
-        setBlueScore(msg.blueScore)
-        setWhiteScore(msg.whiteScore)
-     
-        console.log(msg);
+        msg = JSON.parse(msg.data);
+        setBlueScore(msg.blueScore);
+        setWhiteScore(msg.whiteScore);
       };
     };
-  });
+  }, []);
 
   return (
     <>
       <h1>Smart Kickers</h1>
-      <div className="game-result-container" data-testid="blue-team-score">
-        Blue: {blueScore}
-          {"   "}
-        White: {whiteScore}
+      <div className="game-result-container">
+        <p className="game-result-item">Blue: {blueScore}</p>
+        <p className="game-result-item">White: {whiteScore}</p>
       </div>
-      <center><Button >Reset game</Button></center>
+      <center>
+        <Button>Reset game</Button>
```

Comment:
```suggestion
        <Button onClick={() => socket.send(JSON.stringify(true))}>
          Reset game
        </Button>
```

Reply:
After a short discussion with @marekkawalski, we decided to discard this change due to near communication protocol changes (moving from WebSocket to rest API for no time-critical communication)